packages/emitter-framework/test/typescript/components/interface-declaration.test.tsx

```diff
@@ -674,8 +674,8 @@ describe("Typescript Interface", () => {
         const actualContent = await format(testFile.contents as string, { parser: "typescript" });
         const expectedContent = await format(
           `export interface WidgetOperationsExtended {
-          getName(id: string): Widget;
-          delete(id: string): void;
+          getName: (id: string) => Widget;
+          delete: (id_2: string) => void;
```

Comment:
This one is interesting - does not appear related to my PR but this test was also skipped prior to this. Any idea why the _2 is added here or how to address it would be much appreciated - even a "yeah, this is fine" would be helpful

Reply:
hhm, this I think is a sign that its a missing a scope boundary.
When you create the params you probably want to warp in a `<Scope/>`  component